### PR TITLE
Add functionality for searchBar and fixes

### PR DESCRIPTION
**Pull request by MuhammadAw90**

### CODE REVIEW

**Review by LeenMass — DISMISSED** [review]

**Comment by tawfiknasser** [comment]
@MuhammadAw90 see conflict

**Comment by MuhammadAw90** [comment]
@tawfiknasser , fixed

File: src/pages/Projects/PopUpDialog/index.jsx [hunk 1]
```diff
@@ -27,15 +27,19 @@ export default function PopUpMessage(props) {
...
+          await msgs.current.show[
+            {
+              sticky: false,
+              severity: 'success',
+              summary: '',
+              detail: resultBody.message,
+              closable: true
+            }
+          ],
+          await setTimeout(() => {}, 30000),
```
Comment (tawfiknasser) [inline src/pages/Projects/PopUpDialog/index.jsx]: What this line do ?

Comment (MuhammadAw90) [reply src/pages/Projects/PopUpDialog/index.jsx]: currently nothing, I'm removing it.

**Pull request merged**